pkg/applicationserver/applicationserver.go

```diff
@@ -621,31 +828,31 @@ func (as *ApplicationServer) handleUplink(ctx context.Context, ids ttnpb.EndDevi
 					dev.DevAddr = ids.DevAddr
 					mask = append(mask, "session", "ids.dev_addr")
 					logger.Debug("Restored session")
```

Comment:
This is not necessarily restoration, right? If AS did not receive the join-accept, the uplink may be for a totally new session, which is not initialized

Reply:
I cannot really find a good name here - `Restored previous session` or `Restored external session` seem off here.